libs/application/templates/financial-aid/src/fields/SummaryForm/SummaryForm.tsx

```diff
@@ -0,0 +1,264 @@
+import React from 'react'
+import {
+  Text,
+  Box,
+  GridColumn,
+  GridRow,
+  Button,
+  Icon,
+  Input,
+} from '@island.is/island-ui/core'
+import { useIntl } from 'react-intl'
+import DescriptionText from '../DescriptionText/DescriptionText'
+import { Controller, useFormContext } from 'react-hook-form'
+import * as m from '../../lib/messages'
+import SummaryBlock from '../SummaryBlock/SummaryBlock'
+import { ApproveOptions, FAFieldBaseProps } from '../../lib/types'
+import {
+  Employment,
+  estimatedBreakDown,
+  getNextPeriod,
+  HomeCircumstances,
+} from '@island.is/financial-aid/shared/lib'
+import {
+  getMessageHomeCircumstances,
+  getMessageEmploymentStatus,
+  getMessageApproveOptions,
+  getMessageApproveOptionsForIncome,
+} from '../../lib/formatters'
+import { Breakdown } from '@island.is/financial-aid/shared/components'
+
+const SummaryForm = ({ application, goToScreen }: FAFieldBaseProps) => {
+  const { formatMessage } = useIntl()
+  const { answers, externalData } = application
+
+  const { setValue } = useFormContext()
+
+  const formCommentId = 'formComment'
+
+  return (
+    <>
+      <Box
+        display="flex"
+        alignItems="center"
+        flexWrap="wrap"
+        marginTop={[2, 2, 4]}
+      >
+        <Box marginRight={1}>
+          <Text as="h3" variant="h3">
+            {formatMessage(m.summaryForm.general.descriptionTitle)}
+          </Text>
+        </Box>
+
+        <Text variant="small">
+          {formatMessage(m.summaryForm.general.descriptionSubtitle, {
+            nextMonth: getNextPeriod.month,
+          })}
+        </Text>
+      </Box>
+
+      <Box marginTop={2}>
+        <DescriptionText text={m.summaryForm.general.description} />
+      </Box>
+
+      <Box>{/* TODO estimated aid */}</Box>
+
+      <Box marginTop={2}>
+        <DescriptionText text={m.summaryForm.general.calculationsOverview} />
+      </Box>
+
+      <Box paddingY={[4, 4, 5]} borderTopWidth="standard" borderColor="blue300">
+        <GridRow marginBottom={3}>
+          <GridColumn span={['12/12', '12/12', '12/12', '5/12']}>
+            <Box>
+              <Text fontWeight="semiBold">
+                {formatMessage(m.summaryForm.userInfo.name)}
+              </Text>
+              <Text>{externalData?.nationalRegistry?.data?.fullName}</Text>
```

Comment:
Should not happen, but do we want to display something here and in other places if this is undefined?

Reply:
Yess, do you have any suggestions what should be displayed👀 ?